canal/sync.go

```diff
@@ -128,7 +127,7 @@ func (c *Canal) runSyncBinlog() error {
 				trigger = true
 			} else if mb = checkDropTable(e); mb != nil {
 				schema = e.Schema
-				table = mb[2]
+				table = mb[7]
```

Comment:
I am confused that why 7 here now?

Reply:
yes, you can find this test in canal_test.go , the test function is : TestDropTableExp